Sources/NIOSSL/SSLPrivateKey.swift

```diff
@@ -404,6 +404,19 @@ extension NIOSSLPrivateKey {
             return customKey.signatureAlgorithms
         }
     }
+
+    /// Extracts the bytes of this private key in DER format.
+    /// - Important: If the key is not an `EVP_PKEY`, an empty array will be returned.
+    /// - Returns: The DER-encoded bytes for this `EVP_PKEY` private key.
+    /// - Throws: If an error occurred while serializing the private key.
+    public func toDERBytes() throws -> [UInt8] {
```

Comment:
Can we add tests for this?

Reply:
Welp, I just realise the latest push had failed - tests exist now :D